modules.d/90kernel-modules/module-setup.sh

```diff
@@ -147,4 +147,5 @@ install() {
         inst_hook cmdline 01 "$moddir/parse-kernel.sh"
     fi
     inst_simple "$moddir/insmodpost.sh" /sbin/insmodpost.sh
+    inst_multiple sysctl
```

Comment:
I would recommend to use the "-o" argument here to make it optional dependency.

Reply:
Why? procps-ng is already a hard dependency of dracut anyway, and not having this causes modprobe config files to fail to work if they're using it.

Reply:
Dependency here means a single binary included into the generated initramfs (not a package dependency).

> modprobe config files to fail 
Most examples listed are for nfs. If you want to include systctl as hard dependency (and not an optional one) than perhaps it should be done in the nfs dracut module.

Can you give an example of failure that is not nfs (ideally something that most people would run into) ? 

If an initramfs only boots from local storage (e.g. no networking), why do we need to include a new binary into everybody's generated initramfs ?

Reply:
Vendors might be using systemd only systems in which they so they might want to exclude it thus making it an optional dependency is better as @LaszloGombos is suggesting + we need to properly test this so we dont end up potentially adding some weird race condition with systemd-sysctl encase that module is enabled.

Reply:
Okay, marked as optional now.